### ISSUE TITLE
jsdoc links broken?

### ISSUE THREAD

**Issue by cmonacaps** [freeCodeCamp/devdocs#1034]
In the main, jsdoc api index frame, right hand side [from here](https://devdocs.io/jsdoc/), if I scroll to 
```
   @param (synonyms: @arg, @argument)
      Document the parameter to a function.
```
and click on the link `@param` it sends me [to here](https://devdocs.io/jsdoc/tags-param), where it replaces the right-hand frame content with:
```
                                Page not found.
    It may be missing from the source documentation or this could be a bug.

                                    Go back
```

this worked ok til last week

**Comment by jmerle** [comment]
Do you happen to know whether it was already broken 2 days ago or not?

**Comment by cmonacaps** [comment]
I think yes - I'm pretty sure I noticed it broke last week, i.e. Friday at the latest but maybe last Wednesday.

**Comment by jmerle** [comment]
Odd, 12 minutes ago on my phone (Chrome on Android) I could reproduce this issue, but now I am sitting behind my PC it works just fine on Chrome. Is it still broken after a refresh?

**Comment by cmonacaps** [comment]
Yes just reproduced it again following refresh.  I'm using Chrome on (power book) osx.

**Comment by jmerle** [comment]
Ah, it only seems to work when the doc is enabled first in the sidebar. Seems to be an issue with all disabled docs.

No idea what causes it yet, but I can reproduce it again.

**Comment by jmerle** [comment]
Found the culprit. The service worker prevented requests to the docs' `index.json` file (which contains all path to file mappings) from going through. I deployed a fix in 96cadd449bdc0dbcf0e9cd85288257df2b79e8ef.

To trigger the update you'll need to refresh, wait a while for the "DevDocs has been updated!" notification to appear and refresh two more times for it to start working again.

**Comment by cmonacaps** [comment]
👍 thanks!
